aten/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ set(ATen_CPU_TEST_SRCS)
 set(ATen_CPU_INCLUDE)
 set(ATen_THIRD_PARTY_INCLUDE)
 set(ATen_CUDA_SRCS)
+set(ATen_CUDA_SRCS_SP)
```

Comment:
Nit: Can we make this name more descriptive? ATen_CUDA_SRCS_W_SORT_BY_KEY or something?